File: app/src/main/java/com/hustunique/morii/edit/EditContract.java
package com.hustunique.morii.edit;

import android.content.Context;

public interface EditContract {
    interface IPresenter {
        void getUI();
        void getPicture();

    };
    interface IModel{
        void setAppCompatActivityUse(Context context);

        void getPicture();

    }

    interface IListener{
        void setIt(String path);
    }

    interface IView {

        void setAddPhoto(String path);
    };
}

File: app/src/main/java/com/hustunique/morii/edit/EditContract.kt
package com.hustunique.morii.edit

import android.content.Context

interface EditContract {
    interface IPresenter {
        val picture: Unit
    }

    interface IModel {
        fun setAppCompatActivityUse(context: Context)
        val picture: Unit
    }

    interface IListener {
        fun setIt(path: String)
    }

    interface IView {
        fun setAddPhoto(path: String)
    }
}